Documentation/ApiOverview/GlobalValues/Typo3ConfVars/BE.rst

```diff
@@ -816,6 +816,12 @@ $GLOBALS['TYPO3_CONF_VARS']['BE']['toolbarItems']
    :Default: []
 
    Registered toolbar items classes
+   
+.. note::
+   This configuration variable will be removed with version 12.0. Extensions who want to 
+   support both LTS 11 and 12 can set this variable and depend on Services to load the 
```

Comment:
Can we add more details here? Especially since the `ToolbarItemInterface` already exists and must be implemented anyways.

Reply:
The information about implementing the ToolbarItemInterface does not really belong here, as this is merely a listing of all TYPO3_CONF_VARS. Details about how to add a ToolbarItem would need to go another place and be linked from here. Please fell free to open another PR about this.